tensorflow/contrib/tensorrt/ops/trt_engine_op.cc

```diff
@@ -41,6 +41,7 @@ REGISTER_OP("TRTEngineOp")
     .Attr("workspace_size_bytes: int")
     .Attr("precision_mode: {'FP32', 'FP16', 'INT8', 'INT8CALIB'}")
```

Comment:
I don't understand why both INT8CALIB and use_calibration. Looks like you need either of them not both.

Reply:
Not sure why INT8CALIB argument is there either, it appears to be unused. I will remove it unless we decide to switch to an argument like https://github.com/tensorflow/tensorflow/pull/22788#discussion_r223252646